cli/endpoints/online/managed/vnet/setup/modules/applicationinsights.bicep

```diff
@@ -0,0 +1,30 @@
+// Creates an Application Insights instance as dependency for Azure ML
+@description('Azure region of the deployment')
+param location string
+
+@description('Tags to add to the resources')
+param tags object = {}
+
+@description('Application Insights resource name')
+param applicationInsightsName string
+
+resource applicationInsights 'Microsoft.Insights/components@2020-02-02' = {
+  name: applicationInsightsName
+  location: location
```

Comment:
This will be run only in region other than Canary and westcentralUS right?

Reply:
users can pass appInsightsLocation to the setup/main.bicep to setup appinsights in another region (say eastus)